db-ops/create-db-user.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# docker image with postgres client only
+DOCKER_IMAGE_TAG=governmentpaas/psql:latest
+
+DB_ENDPOINT=$(aws rds describe-db-instances --region=$REGION --query "DBInstances[?DBName=='$PROJECT_NAME'].Endpoint.Address" | jq -r '.[0]')
+SECRET_ID=$(aws secretsmanager list-secrets --region $REGION  --query "SecretList[?Name=='$PROJECT_NAME-$ENVIRONMENT-rds-$SEED'].Name" | jq -r ".[0]")
+# RDS MASTER
+MASTER_RDS_USERNAME=master_user
+SECRET_PASSWORD=$(aws secretsmanager get-secret-value --region=$REGION --secret-id=$SECRET_ID | jq -r ".SecretString")
+# APPLICATION DB ADMIN
+DB_APP_USERNAME=$PROJECT_NAME
+DB_APP_PASSWORD=$(LC_ALL=C tr -dc 'A-Za-z0-9' < /dev/urandom | base64 | head -c 16)
+
+# Fill in env-vars to db user creation manifest
+eval "echo \"$(cat ./db-ops/job-create-db.yml.tpl)\"" > ./k8s-job-create-db.yml
+# the manifest creates 4 things
+# 1. Namespace: db-ops 
+# 2. Secret in db-ops: db-create-users (with master password, and a .sql file
+# 3. Job in db-ops: db-create-users (runs the .sql file against the RDS given master_password from env)
+# 4. Secret in Application namespace with DB_USERNAME / DB_PASSWORD 
+kubectl create -f ./k8s-job-create-db.yml
+
+# Deleting the entire db-ops namespace, leaving ONLY application-namespace's secret behind
+kubectl wait --for=condition=complete --timeout=10s job db-create-users
+if [ $? -eq 0 ]
+then 
+  kubectl delete namespace db-ops
+else 
+  POD_NAME=$(kubectl -n db-ops get po | grep db-create-users |awk '{print $1}')
```

Comment:
You could clean this up a bit by giving the pod spec in the job a label like `app: db-create-users` and then a user could run `kubectl logs -n db-ops -l app=db-create-users`

Reply:
oh nice, didnt know it supports labels 👁️

Reply:
looks like they come with a job-name label, gonna use that instead 
![image](https://user-images.githubusercontent.com/1787590/86647373-fe9a6480-bfad-11ea-9b78-09a911261953.png)